modules/lib/src/main/resources/assets/js/app/view/context/widget/version/VersionHistoryListItem.ts

```diff
@@ -33,11 +33,11 @@ export class VersionHistoryListItem
     private createVersionViewer(): VersionHistoryListItemViewer {
         const versionViewer: VersionHistoryListItemViewer = new VersionHistoryListItemViewer();
 
-        if (this.version.isRevertable() || this.version.isActive()) {
+        if (this.isInteractableItem()) {
             this.addOnClickHandler(versionViewer);
         }
 
-        if (this.version.isRevertable()) {
+        if (this.isInteractableItem()) {
```

Comment:
Any reason for not joining these two `if`s?

Reply:
Missed it, will fix